### DISCUSSION TITLE
Age rating settings "Lootboxes" - how to set in fastlane?

### DISCUSSION THREAD

**Discussion by anh1979** [fastlane/fastlane#22175] [Q&A]
Hey there,

I just spotted that new setting in AppStoreConnect this morning  and I wonder what to add to my definition to set that correctly for AppStoreConnect. Does anybody already know?

See here the dialog window, where you find the new option (age rating, step 2):
<img width="616" alt="Bildschirmfoto 2024-08-06 um 10 11 10" src="https://github.com/user-attachments/assets/9416bfce-2525-4002-aef8-5c7353712684">

Appreciate any help. Up to now, the API doesn't throw any errors on this at my place.

Best regards,
Anne

**Comment by anh1979** [comment]
When I try to add "lootBox":0 to my rating.json (the input field in AppSoreConnect is named so), the API tells me =>

"The provided entity includes an unknown attribute - 'lootBox' is not an attribute on the resource 'ageRatingDeclarations'"

So, I guess I have to wait for it...

**Comment by gholias** [comment]
I'm also looking for that information

**Comment by andrewdieken** [comment]
Looks like there's an open PR adding support for the new loot box age rating declaration https://github.com/fastlane/fastlane/pull/22177
